<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stackedwidget</class>
 <widget class="QStackedWidget" name="stackedwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="page_systemState">
   <widget class="QWidget" name="widget_systemState" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>520</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>520</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>520</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_device_number">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_1background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>520</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/xitongjiance_01.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_system_id">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_running_status">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>150</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_sensor_status">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_camera_status">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_tf_status">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>300</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_positioning_status">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>350</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_system_booting">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_1background</zorder>
    <zorder>label_device_number</zorder>
    <zorder>label_system_id</zorder>
    <zorder>label_running_status</zorder>
    <zorder>label_sensor_status</zorder>
    <zorder>label_camera_status</zorder>
    <zorder>label_tf_status</zorder>
    <zorder>label_positioning_status</zorder>
    <zorder>label_system_booting</zorder>
   </widget>
   <widget class="QWidget" name="widget_biaoti1systemState" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/biaoti_01.bmp);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_workingState">
   <widget class="QWidget" name="widget_biaoti2workingState" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/biaoti_02.bmp);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_workingState" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_2background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>520</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/gongzuojianshi_01.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_depth">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_y_data">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_running_status2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>170</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_expected_area">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_tf_storage">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>220</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_tota_area">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>270</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_x_data">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_oil">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>220</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="page_cammeraDisplay">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <widget class="QWidget" name="widget_cammeraDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>800</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_mapNavigation">
   <widget class="QWidget" name="widget_biaoti4mapNavigation" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QRadioButton" name="hybridButton">
     <property name="geometry">
      <rect>
       <x>709</x>
       <y>0</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Satellite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/biaoti_04.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_3background</zorder>
    <zorder>hybridButton</zorder>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>520</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_startDisplay">
   <widget class="QWidget" name="widget_biaoti6startDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/biaoti_06.bmp);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_startDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_5background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>520</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/kaiji_01.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
